### PR TITLE
factorio magnetism

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request
Adds a new factorio magnetism spell that when holding church shields, healing items, and weapons (not implanted) become no drop.
Does not work for armor/cells/guns as that would be either useless or op

### CODE REVIEW

File: code/modules/core_implant/cruciform/rituals/path.dm [hunk 1]
```diff
@@ -933,3 +933,65 @@
...
+					for(var/obj/item/organ_module/active/simple/OM in arm.contents)
+						if(OM.holding == O)
+							not_implanted = FALSE
+				if(not_implanted)
+					O.canremove = !O.canremove
+					success = TRUE
+					break
+				else
+					//Prevents teleportation implants.
+					fail("[O.name] is apart of an implant, making this pointless.", user, C)
```
Comment (Smokingsquirrel) [inline code/modules/core_implant/cruciform/rituals/path.dm:979]:
```suggestion
					fail("[O.name] is part of an implant, making this pointless.", user, C)
```

File: code/modules/core_implant/cruciform/rituals/path.dm [hunk 2]
```diff
@@ -933,3 +933,65 @@
...
+/datum/ritual/cruciform/factorial/nodrop_magnet
+	name = "Hand of Magnetism"
+	phrase = "Ager attractivus operis mei ad se alligavit." //"Field of attraction bound my work to self."
+	desc = "Try to make whatever you are holding no longer droppable. \
+	Only works with church-based items. \
+	If used on an item that already can't be dropped, it will become droppable. \
+	Does not work with firearms or cells."//To op to have a no drop on command gun sadly
+	power = 15
+	cooldown = FALSE
+	success_message = "Whatever item that you are holding feels impossible to let go."
```
Comment (Smokingsquirrel) [inline code/modules/core_implant/cruciform/rituals/path.dm:946]:
```suggestion
	success_message = "The item that you are holding feels impossible to escape the Physics of the Absolute.“
```

File: code/modules/core_implant/cruciform/rituals/path.dm [hunk 3]
```diff
@@ -933,3 +933,65 @@
 		return FALSE
 	anti_cheat = FALSE
 	return TRUE
+
+/datum/ritual/cruciform/factorial/nodrop_magnet
+	name = "Hand of Magnetism"
+	phrase = "Ager attractivus operis mei ad se alligavit." //"Field of attraction bound my work to self."
+	desc = "Try to make whatever you are holding no longer droppable. \
```
Comment (Smokingsquirrel) [inline code/modules/core_implant/cruciform/rituals/path.dm:940]:
```suggestion
	desc = "Harness the Physics of the Absolute. Items sanctified by the Church you are holding are no longer droppable. \
```

**Review by Smokingsquirrel — COMMENTED** [review]
Suggestion to Churchify the litany with different wording to make it feel less on the nose.

**Comment by Smokingsquirrel** [comment]
Factorio

IMG_3929

File: code/modules/core_implant/cruciform/rituals/path.dm [hunk 4]
```diff
@@ -933,3 +933,65 @@
...
+/datum/ritual/cruciform/factorial/nodrop_magnet
+	name = "Hand of Magnetism"
+	phrase = "Ager attractivus operis mei ad se alligavit." //"Field of attraction bound my work to self."
+	desc = "Harness the Physics of the Absolute. Items sanctified by the Church you are holding are no longer droppable. \
+	Only works with church-based items. \
+	If used on an item that already can't be dropped, it will become droppable. \
+	Does not work with firearms or cells."//To op to have a no drop on command gun sadly
+	power = 15
+	cooldown = FALSE
+	success_message = "The item that you are holding feels impossible to escape the Physics of the Absolute.“
```
Comment (Trilbyspaceclone) [inline code/modules/core_implant/cruciform/rituals/path.dm:946]:
```suggestion
	success_message = "The item that you are holding feels impossible to escape the Physics of the Absolute."
```

**Pull request merged**